Utils/Make.PeleLMeX

```diff
@@ -25,6 +25,23 @@ ifeq ($(USE_EFIELD),TRUE)
    DEFINES += -DPELE_USE_EFIELD
    LMdirs += Source/Efield Source/Efield/GMRES Source/Efield/LinOps
 endif
+ifeq ($(USE_PARTICLES), TRUE)
+   ifdef SPRAY_FUEL_NUM
+      DEFINES += -DSPRAY_PELE_LM
```

Comment:
I'm guessing we can't easily change those to be more uniform with other defines (SPRAY_PELE_LM -> PELE_USE_SPRAY) ?